### PR TITLE
Version bumps + build fixups

### PR DESCRIPTION
**Pull request by ArturGajowy**
## Overview
<sup>_What this PR does, and why it's needed_</sup>

Version bumps for scala and cats-effect.

I've also noticed a setting I've enabled a while ago could lead to unexpected contents of produced artifacts. The last commit improves on that.

**EDIT** see commit log for details :)

### JIRA ticket:
<sup>_Create it if there isn't one already._</sup>

### Notes
<sup>_Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else._</sup>

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

**Comment by ArturGajowy** [comment]
bors try

File: build.sbt [hunk 1]
```diff
@@ -37,6 +37,15 @@ lazy val projectSettings = Seq(
...
   IntegrationTest / testForkedParallel := false,
+  assemblyMergeStrategy in assembly := {
+    // For some reason, all artifacts from 'io.netty' group contain this file with different contents.
+    // Discarding it as it's not needed.
+    case path if path.endsWith("io.netty.versions.properties") => MergeStrategy.discard
+    // The scala compiler includes native bindings for jansi under the same path jansi does.
+    // This should pick the ones provided by jansi.
+    case path if path.startsWith("META-INF/native/") && path.contains("jansi") => MergeStrategy.last
+    case path => MergeStrategy.defaultMergeStrategy(path)
+  }
```
Comment (KentShikama) [inline build.sbt]: I thought we no longer use assembly?

Comment (KentShikama) [reply build.sbt]: Ok

**Comment by ArturGajowy** [comment]
bors try

**Pull request merged**